ComposeGallery/src/main/java/com/microsoft/device/display/samples/composegallery/ui/view/ListPane.kt

```diff
@@ -79,27 +75,23 @@ private fun ListActions() {
 @Composable
 private fun GalleryList(
     models: List<ImageModel>,
-    isDualMode: Boolean,
     selectedImageIndex: Int,
     updateImageIndex: (Int) -> Unit,
-    lazyListState: LazyListState,
 ) {
     LazyColumn(
         modifier = Modifier
             .fillMaxSize()
             .testTag(stringResource(R.string.gallery_list)),
-        state = lazyListState,
```

Comment:
I'm just curious - why did you remove the lazy list state?

Reply:
I didn't see any customized need for creating our own `lazyListState` here. By default, `LazyColumn` provides a `LazyListState` via `state: LazyListState = rememberLazyListState()`.

Reply:
Ahh ok makes sense, thank you for the explanation!